src/tests/dotnet-trace/DotnetTrace.UnitTests.csproj

```diff
@@ -1,12 +1,12 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>net5.0</TargetFramework>
+    <TargetFramework>net6.0</TargetFramework>
   </PropertyGroup>
 
   <ItemGroup>
-    <ProjectReference Include="../../Tools/dotnet-trace/dotnet-trace.csproj" />
-    <ProjectReference Include="..\ExitCodeTracee\ExitCodeTracee.csproj" />
+    <ProjectReference Include="..\..\Tools\dotnet-trace\dotnet-trace.csproj" />
+    <ProjectReference Include="..\CommonTestRunner\CommonTestRunner.csproj" />
     <ProjectReference Include="..\Microsoft.Diagnostics.NETCore.Client\Microsoft.Diagnostics.NETCore.Client.UnitTests.csproj" />
```

Comment:
Do you know why these tests depend on client tests?

Reply:
The dotnet trace tests references code in the dotnet-trace itself.  I tried to remove it but the tests don't build.

Reply:
I mean, why are dotnet trace unit tests depending on the client unit tests?

Reply:
It looks like it doesn't need the reference. I will removed it.